docs/SRS-Volere/SRS.tex

```diff
@@ -905,7 +907,8 @@ \subsection{Adaptability Requirements}
     Microsoft Edge browsers.
   \item The application must be able to run on tablets, smartphones, and
     laptops.
-  \item The application must be able to run on Android, IOS, and Windows 10
+  \item \sout{The application must be able to run on Android, IOS,
```

Comment:
This requirement isn't relevant. This is a website, so OS shouldn't have much of an impact, presuming that the browsers have no bugs, which I think is a safe assumption.